src/resolve/resolved-conf.c

```diff
@@ -27,11 +28,12 @@ static int manager_add_dns_server_by_string(Manager *m, DnsServerType type, cons
         union in_addr_union address;
         int family, r, ifindex = 0;
         DnsServer *s;
+        _cleanup_free_ char *server_name;
```

Comment:
`= NULL` missing. This will crash if in_addr_ifindex_name_from_string_auto() fails.